pdchaosazure/vmss/actions.py

```diff
@@ -37,22 +39,29 @@ def delete_instance(filter: str = None,
     logger.debug(
         "Starting {}: configuration='{}', filter='{}'".format(delete_instance.__name__, configuration, filter))
 
-    vmss = fetch_vmss(filter, configuration, secrets)
+    client = init_client(secrets, configuration)
+    vmss_list = fetch_vmss(filter, configuration, secrets)
     vmss_records = Records()
-    for set in vmss:
+
+    for vmss in vmss_list:
         instances_records = Records()
-        instances = fetch_instances(set, instance_criteria, configuration, secrets)
+        instances = fetch_instances(vmss, instance_criteria, configuration, secrets)
```

Comment:
This guy should take client as well instead of initializing it one more time

Reply:
done